### PR TITLE
{bio}[foss/2022a] GOATOOLS v1.3.1, XlsxWriter v3.0.8 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3135.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/b57953489231fe4c00e6229169c86a75 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Comment by casparvl** [comment]
Test report by @casparvl
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in total)
tcn309.local.snellius.surf.nl - Linux Rocky Linux 8.7, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.6.8
See https://gist.github.com/53fee3ffc1886c5ed33f49babee22f1f for a full test report.

File: easybuild/easyconfigs/g/GOATOOLS/GOATOOLS-1.3.1-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+version = '1.3.1'
+
+homepage = 'https://github.com/tanghaibao/goatools'
+description = "A Python library for Gene Ontology analyses"
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+source_urls = ['https://github.com/tanghaibao/goatools/archive/']
+sources = [{
+    'git_config': {
```
[the remark this answers is not in the document]
Comment (boegel) [reply easybuild/easyconfigs/g/GOATOOLS/GOATOOLS-1.3.1-foss-2022a.eb:16]: clarified in comment added in 73ba40dbfb

**Review by casparvl — APPROVED** [review]
All good now, ready to merge!

**Comment by casparvl** [comment]
Going in, thanks @boegel!

**Pull request merged**